opentelemetry-sdk/src/opentelemetry/sdk/propagator/b3_format.py

```diff
@@ -92,7 +93,11 @@ def extract(cls, get_from_carrier, carrier):
         if sampled in cls._SAMPLE_PROPAGATE_VALUES or flags == "1":
             options |= trace.TraceOptions.RECORDED
 
-        return trace.SpanContext(
+        # TODO: there is no standard way to retrieve
+        # the SpanContext or DistributedContext from a
+        # context object declared in opentelemetry-api.
+        # this should be more standard.
+        context[Tracer.CONTEXT_SLOT_NAME] = trace.SpanContext(
```

Comment:
This is wrong. In the context there is always a full span, not just a context (see Tracer.get_current_span). The parent span context should never make it to the context.

Reply:
Will take a look.